proto/decentraland/sdk/components/ui_background.proto

```diff
@@ -2,10 +2,22 @@
 package decentraland.sdk.components;
 
 import "decentraland/sdk/components/common/id.proto";
-option (common.ecs_component_id) = 1053;
+option (common.ecs_component_id) = 1092;
 
 import "decentraland/common/colors.proto";
+import "decentraland/common/texture.proto";
+import "decentraland/common/rect.proto";
 
 message PBUiBackground {
-  optional decentraland.common.Color4 background_color = 1; // default=(0.0, 0.0, 0.0, 0.0)
+  optional decentraland.common.Color4 background_color = 1; // default=(0.0, 0.0, 0.0, 0.0) 
+  optional decentraland.common.Texture texture = 2;
+  BackgroundTextureMode texture_mode = 3;
+  optional decentraland.common.Rect texture_slices = 4; // default=(1/3, 1/3, 1/3, 1/3)
+  optional decentraland.common.Rect uv = 5;
+}
+
+enum BackgroundTextureMode {
+  NINE_SLICES = 0;
+  CENTER = 1;
+  STRETCH = 2;
```

Comment:
Please copy the comments from the issue, those will generate documentation in the generated code